_includes/header.html

```diff
@@ -6,7 +6,7 @@
             </button>
             <a class="navbar-brand page-scroll" href="/#page-top">
                 <span class="logo">
-                    <img src="//opensourcedesign.net/images/opensourcedesign-logo.svg" class="logo" alt="Open Source Design Logo"/>
+                    <img src="//opensourcedesign.net/images/opensourcedesign-logo.svg" alt="Open Source Design Logo"/>
```

Comment:
We don’t need this alt text, as the logo is purely cosmetic. It says "Open Source Design" in the next line, so we can use `alt=""` here. :)